input/resources/au-erequesting-placer.xml

```diff
@@ -83,5 +83,252 @@
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- PRACTITIONERROLE -->
+    <resource>
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="SHOULD"/>
+      </extension>
+      <type value="PractitionerRole"/>
+      <supportedProfile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitionerrole">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Placers **SHOULD** support the PractitionerRole resource, the AU Core profile, and the conformance expectations for the PractitionerRole resource."/>
+      <!-- TBC 
+      <referencePolicy value="resolves">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- CONDITION -->
+    <resource>
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="MAY"/>
+      </extension>
+      <type value="Condition"/>
+      <supportedProfile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-condition">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Placers **MAY** support the Condition resource, the AU Core profile, and the conformance expectations for the PractitionerRole resource."/>
+      <!-- TBC 
+      <referencePolicy value="resolves">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- DOCUMENTREFERENCE -->
+    <resource>
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="MAY"/>
+      </extension>
+      <type value="DocumentReference"/>
+      <supportedProfile value="http://hl7.org/fhir/StructureDefinition/DocumentReference">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="MAY"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Placers **MAY** support the DocumentReference resource."/>
+      <!-- TBC 
+      <referencePolicy value="resolves">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHOULD"/>
+        </extension>
+      </referencePolicy> -->
+    </resource>
+    <!-- ENCOUNTER -->
```

Comment:
Encounter included twice causes: Constraint failed: cpb-9: 'A given resource can only be described once per RESTful mode.
Remove this entry (MAY expectation).

Reply:
The erronous inclusion of Encounter as MAY is removed.